internal/config/config.go

```diff
@@ -42,6 +43,10 @@ type Config struct {
 	LogVerbose bool
 }
 
+func Defaults() Config {
+	return ConfigDefaults
```

Comment:
If you provide a function to get the default config, I think you can hide `ConfigDefaults`. Otherwise, it might be confusing and `ConfigDefaults` might be used incorrectly.